requirements_all.txt

```diff
@@ -2573,6 +2573,12 @@ uvcclient==0.11.0
 # homeassistant.components.vallox
 vallox-websocket-api==3.0.0
 
+# homeassistant.components.vartastorage
+vartastorage==0.2.1
```

Comment:
This seems to be a remnant from previous merges

Reply:
fixed - thx